Index: yade/CMakeLists.txt
===================================================================
--- yade.orig/CMakeLists.txt
+++ yade/CMakeLists.txt
@@ -730,7 +730,7 @@ IF(REAL_USE_ARBITRARY EQUAL "ON")
 ENDIF()
 
 # On clang and other architectures the boost::float128 type is not available. The RealHP<N> still works! It simply uses MPFR.
-IF((${ARCHITECTURE} STREQUAL "arm64") OR (${ARCHITECTURE} STREQUAL "mips64el") OR (${ARCHITECTURE} STREQUAL "ppc64el") OR (${ARCHITECTURE} STREQUAL "s390x") OR ("${CMAKE_CXX_COMPILER} ${CMAKE_CXX_COMPILER_ARG1}" MATCHES ".*clang"))
+IF((${ARCHITECTURE} STREQUAL "arm64") OR (${ARCHITECTURE} STREQUAL "mips64el") OR (${ARCHITECTURE} STREQUAL "ppc64el") OR (${ARCHITECTURE} STREQUAL "s390x") OR (${ARCHITECTURE} STREQUAL "riscv64") OR (${ARCHITECTURE} STREQUAL "loong64") OR ("${CMAKE_CXX_COMPILER} ${CMAKE_CXX_COMPILER_ARG1}" MATCHES ".*clang"))
   SET(MARK_FLOAT128_AVAILABLE "OFF")
   ADD_DEFINITIONS("-DYADE_FLOAT128_UNAVAILABLE")
 ELSE()
